### PR TITLE
Update evaluate-attack-surface-reduction.md

### PR DESCRIPTION
**Pull request by kurtsarens**
adding Warning for Server 2016 as ASR rules can have unexpected results on this unsupported platform

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/security/defender-endpoint/evaluate-attack-surface-reduction.md [hunk 1]
```diff
@@ -35,6 +35,9 @@ Attack surface reduction rules help prevent actions typically used by malware to
 
 Learn how to evaluate attack surface reduction rules by enabling audit mode to test the feature directly in your organization.
 
+> [!Warning]
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/evaluate-attack-surface-reduction.md]:
```suggestion
> [!WARNING]
```

File: microsoft-365/security/defender-endpoint/evaluate-attack-surface-reduction.md [hunk 2]
```diff
@@ -35,6 +35,9 @@ Attack surface reduction rules help prevent actions typically used by malware to
...
+> Enabling Attack Service Reduction rules on Windows Server 2016 may lead to unexpected results and impact server performance. It is not recommended to enable or deploy Attack Surface Reduction rules to unsupported platforms.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/evaluate-attack-surface-reduction.md]:
```suggestion
> Enabling attack service reduction rules on Windows Server 2016 may lead to unexpected results and impact server performance. We do not recommend enabling or deploying attack surface reduction rules to unsupported platforms.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
@kurtsarens - please also change the word "Warning" to "WARNING" ( all caps) as indicated. Thanks.

**Pull request merged**